src/OpenTelemetry.Api/Trace/SpanContext.cs

```diff
@@ -121,6 +121,17 @@ public bool IsRemote
             }
         }
 
+        /// <summary>
+        /// Gets the <see cref="TraceStateString"/> associated with this <see cref="SpanContext"/>.
+        /// </summary>
+        public string TraceStateString
```

Comment:
Added a new property, so we won't need to keep converting from ienumerable to string.

Reply:
Seems like a good add to me!